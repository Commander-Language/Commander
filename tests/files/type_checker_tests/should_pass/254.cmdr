//Test that lambdas type check
x = (fn (a, b) -> a + b;)(3, 4);
