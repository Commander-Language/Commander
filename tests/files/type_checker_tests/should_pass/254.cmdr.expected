(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode x) SET (CallExprNode (LambdaExprNode (BindingsNode (BindingNode a) (BindingNode b)) (ReturnStmtNode (BinOpExprNode (LValueExprNode (VarExprNode a)) ADD (LValueExprNode (VarExprNode b)))) (fn (ANY, ANY) -> ANY)) (ExprsNode (IntExprNode 3 INT) (IntExprNode 4 INT))))))
